source/base/utilities.cc

```diff
@@ -13,6 +13,11 @@
 //
 // ---------------------------------------------------------------------
 
+#include <deal.II/base/config.h>
+
+#ifdef DEAL_II_MSVC
+#  include <winsock2.h>
+#endif
```

Comment:
Would you please add this description as a comment for why we do this in the file?